Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -285,12 +285,12 @@ extension NavigationMapView {
 
                         var lineStrings: [LineString] = []
                         
-                        if queriedFeature.feature.geometry.geometryType == GeometryType_Line,
-                           let coordinates = queriedFeature.feature.geometry.extractLocationsArray() as? [CLLocationCoordinate2D] {
-                            lineStrings.append(LineString(coordinates))
-                        } else if queriedFeature.feature.geometry.geometryType == GeometryType_MultiLine,
-                                  let coordinates = queriedFeature.feature.geometry.extractLocations2DArray() as? [[CLLocationCoordinate2D]] {
-                            for coordinates in coordinates {
+                        if queriedFeature.feature?.geometry.type == .LineString,
+                           let lineString = queriedFeature.feature?.geometry.value as? LineString {
+                            lineStrings.append(lineString)
+                        } else if queriedFeature.feature?.geometry.type == .MultiLineString,
```

Comment:
Use a switch/case statement for clarity and better type safety.

This change means that the generated Objective-C headers now use enumerations to differentiate types within a variant. We should take advantage of that change by introducing switch/case statements elsewhere, such as the RoadObjectLocation initializer that was patched in #3250. Look for matches of the regular expression `if\s+[^{]+\.is\w+\(\)` in the codebase.

Reply:
Agree. Changes, which apply switch/case instead of if/else statements were applied to `RoadObjectLocation.swift`, `OrnamentsController.swift` and `DistancedRoadObject.swift`.